package.json

```diff
@@ -47,7 +47,14 @@
     "decimal.js": "^7.1.1",
     "dotenv": "^4.0.0",
     "expand-tilde": "^2.0.2",
+    "express": "^4.15.3",
```

Comment:
I'm not sure if this line is meant to be here... As far as I am aware we do not need this package for the SDK.